frontend/internal-packages/e2e/tests/vrt/vrt.test.ts

```diff
@@ -1,12 +1,22 @@
 import type { Page } from '@playwright/test'
 import { expect, test } from '@playwright/test'
 
-const screenshot = async (page: Page, targetPage: TargetPage) => {
-  await page.goto(targetPage.path)
+const waitForPageReady = async (page: Page) => {
+  await page.waitForLoadState('domcontentloaded')
+  await page.waitForLoadState('load')
+  await page.waitForLoadState('networkidle')
+  await page.evaluate(() => document.fonts.ready)
```

Comment:
Add a process that waits for the page content to load.
This did not affect this issue, but we will leave it added to contribute to long-term stability.